packages/agent-core/src/managers/tool-manager.ts

```
@@ -3,181 +1,61 @@
 import type { CoreMessage } from "ai";
 
 // Constants
 const LOG_PREFIX = "[AgentCore]";
 
 export class ToolManager implements IToolManager {
   private conversationHistory: CoreMessage[] = [];
-  private cachedTools: any = null;
-  private cachedFormattedTools: string | null = null;
 
-  constructor(private mcpService?: IMCPConnectionService) {}
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)